src/ape_cache/_cli.py

```diff
@@ -0,0 +1,46 @@
+import click
+import pandas as pd
+from sqlalchemy import create_engine  # type: ignore
+
+from ape.cli import NetworkBoundCommand, network_option
+from ape.logging import logger
+from ape.utils import ManagerAccessMixin
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Command-line helper for managing query caching database.
+    """
+
+
+@cli.command(cls=NetworkBoundCommand, short_help="Initialize a new cache database")
```

Comment:
Think might have to talk more about this

Reply:
I am closing this conversation because it is stale